ch05construction/02conventions.ipynb

```diff
@@ -386,6 +434,63 @@
     "It is a good idea to run a linter before every commit, or include it in your CI tests.\n",
     "\n"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "There are other tools that help with linting that are worth mentioning.\n",
+    "With `pylint` you can also get other useful information about the quality of your code:\n",
+    "\n",
+    "`pip install` [pylint](https://www.pylint.org/)\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "************* Module species\n",
+      "species.py:2:0: W0311: Bad indentation. Found 5 spaces, expected 4 (bad-indentation)\n",
+      "species.py:1:0: C0111: Missing module docstring (missing-docstring)\n",
+      "species.py:1:0: C0111: Missing class docstring (missing-docstring)\n",
+      "species.py:1:0: R0205: Class 'Species' inherits from object, can be safely removed from bases in python3 (useless-object-inheritance)\n",
+      "species.py:1:0: R0903: Too few public methods (0/2) (too-few-public-methods)\n",
+      "\n",
+      "-------------------------------------\n",
+      "Your code has been rated at -15.00/10\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "%%bash\n",
+    "pylint species.py"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "and with [black](https://black.readthedocs.io/) you can fix all the errors at once.\n",
+    "```bash\n",
+    "black species.py\n",
+    "```\n",
+    "These linters can be configured to choose which points to flag and which to ignore.\n",
+    "\n",
+    "Do not blidnly believed all these automated tools! Style guides are **guides** not **rules**."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "[editorconfig](https://editorconfig.org/) helps to share conventions within a project with multiple contributors, where each of them uses different IDEs and tools. Where bots like [pep8speaks](https://pep8speaks.com/) provides informations to contributors' Pull Requests on which things are not following the conventions for the project.\n"
```

Comment:
Second sentence ("Where bots like...") seems to be missing something - maybe it should be connected to the previous one?